earth_enterprise/src/server/gestream_tables.sql

```diff
@@ -115,13 +115,19 @@ BEGIN
     alter table target_db_table add primary key (target_id, db_id);
   END IF;
 
-  IF NOT check_table_exists('publish_context_table') THEN
+
+  IF check_table_exists('publish_context_table') THEN
+    IF NOT EXISTS(select column_name from information_schema.columns where table_name = 'publish_context_table' and column_name = 'ec_default_ge') THEN
```

Comment:
Design (minor): It seems like it may not be the most efficient use of space to add a column to all records in a table, if only one of the values in the whole column can be `true`.  It may be more efficient to store the ID of a default database separately.

If you use the boolean column, you can also add a partial index on it, so that querying for the row a value of `true` in the column doesn't require iterating over the whole table.

I don't know how many databases common users have, but my guess is that they may not be more than 1000s, in which case, the performance hit may not be significant.